code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1133,7 +1132,7 @@
 		/mob/living/simple_animal/hostile/mimic/crate,
 		/mob/living/simple_animal/hostile/mimic/crate/chest,
 		/mob/living/simple_animal/hostile/mimic/crate/item,
-		) + typesof(/mob/living/simple_animal/hostile/humanoid) + typesof(/mob/living/simple_animal/hostile/asteroid) //Exclusion list for things you don't want the reaction to create.
```

Comment:
The last time I witnessed someone with a goldslime grenade it took half an hour to clean the place out and at least five people died. They're crazy strong as-is.